python/pyhail/keytable.py

```diff
@@ -50,7 +51,7 @@ def key_names(self):
         :rtype: list of str
         """
         try:
-            return self.jkt.keyNames()
+            return list(self.jkt.keyNames())
```

Comment:
Interesting.  That object is still list-like:

```
>>> x = jarray(hc.gateway, hc.jvm.java.lang.String, ['foo', 'bar'])
>>> x
JavaObject id=o23
>>> x[0]
u'foo'
>>> len(x)
2
```

Obviously I like your way better.